TwoLayers_Geometry.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Oct 11 15:56:57 2024
+
+@author: Anupama Rajendra
+"""
+
+import openmc
+
+# Create geometry
+#Spherical shell:
+def make_spherical_shell(inner_radius_W, outer_radius_W, inner_radius_C, M_1, M_2):    
+    S_W_1= openmc.Sphere(r=inner_radius_W) #sphere of radius 1000cm
+    inside_W_sphere_1 = -S_W_1
+    outside_W_sphere_1 = +S_W_1
+    S_W_2 = openmc.Sphere(r=outer_radius_W, boundary_type='vacuum')
+    inside_W_sphere_2 = -S_W_2
+    outside_W_sphere_2 = +S_W_2
+    S_W_3 = outside_W_sphere_1 & inside_W_sphere_2 #filled with specified material

```

Comment:
`S_W_1` and `S_W_2` are surfaces, but `S_W_3` is a region - better variable names will make this more readable.

It's also not clear that you need to define the intermediate variables `inside_...` and `outside_...`.  If `S_W_1` and `S_W_2` (and `S_C_1` below, etc) were named better, then the OpenMC notation with `-` and `+` might just be clear
```suggestion
    inner_W_sphere= openmc.Sphere(r=inner_radius_W) #sphere of radius 1000cm
    outer_W_sphere = openmc.Sphere(r=outer_radius_W, boundary_type='vacuum')
    W_region = +inner_W_sphere & -outer_W_sphere
```